nifi-registry/nifi-registry-core/nifi-registry-web-api/pom.xml

```diff
@@ -467,11 +467,6 @@
             <version>3.2.1</version>
             <scope>test</scope>
         </dependency>
-        <dependency>
```

Comment:
[WARNING] Some problems were encountered while building the effective model for org.apache.nifi.registry:nifi-registry-web-api:war:1.14.0-SNAPSHOT
[WARNING] 'dependencies.dependency.(groupId:artifactId:type:classifier)' must be unique: org.spockframework:spock-core:jar -> duplicate declaration of version (?) @ line 480, column 21